plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestMetricsWrapper.java

```diff
@@ -46,7 +46,7 @@ public void testRoundTrip()
         Map<Integer, ByteBuffer> lowerBounds = ImmutableMap.of(13, ByteBuffer.wrap(new byte[] {0, 8, 9}));
         Map<Integer, ByteBuffer> upperBounds = ImmutableMap.of(17, ByteBuffer.wrap(new byte[] {5, 4, 0}));
 
-        Metrics expected = new Metrics(recordCount, columnSizes, valueCounts, nullValueCounts, lowerBounds, upperBounds);
+        Metrics expected = new Metrics(recordCount, columnSizes, valueCounts, nullValueCounts, ImmutableMap.of(), lowerBounds, upperBounds);
```

Comment:
Add values so we test the serialization

Reply:
We should also test NaN counts for Parquet

Reply:
I don't see any tests for metrics for Parquet format. Nothing like `TestIcebergOrcMetricsCollection` exists for parquet. I will add a ticket to create such a test

Reply:
https://github.com/trinodb/trino/issues/11182